articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
 uid: how-to-guides.create-an-application-template
 ---
 # Create an Application Template
+
+Do you find yourself creating new applications in Intent Architect by choosing a very simple application template and customizing it by installing lots of modules?
+Working in a small team or organization can become daunting when someone else needs to create a new application and are not sure what the configuration settings should be.
+
+In this how-to we will explain how to setup your own team or organizational application template list which can run side-by-side to the official application template list and allows developers not only to create their own applications with ease, but they can easily customize it without having too much in-depth knowledge upon creation.
+
+In Intent Architect, choose to create a new application and select the `Intent Application Template Builder`.
+Give it the name of `MyApplicationTemplates`.
```

Comment:
Add a line above this to make a new paragraph, or alternatively remove the line break.

Perhaps change this to:
> Give it a name such as `MyApplicationTemplates`.